keiko-redis/src/main/kotlin/com/netflix/spinnaker/q/redis/RedisQueue.kt

```diff
@@ -290,7 +292,8 @@ class RedisQueue(
       score().toString(),
       10.toString(), // TODO rz - make this configurable.
       lockTtlSeconds.toString(),
-      score(ackTimeout).toString()
+      format("%f", score(ackTimeout)),
+      format("%f", score())
```

Comment:
Scores (defaultAckTimeout, baseAckTimeout) need to be passed as floats to lua so I can correctly run some math on them.